charts/velero/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.5.3
 description: A Helm chart for velero
 name: velero
-version: 2.15.0
+version: 2.15.1
```

Comment:
```suggestion
version: 2.16.0
```
because it is a feature addition and not a fix
besides that LGTM

Reply:
Since I was trying to fix an issue so I think it's a fix.

Reply:
Never mind, agree with you.